synapse/tests/test_cortex.py

```diff
@@ -7992,6 +7992,10 @@ async def test_cortex_ext_httpapi(self):
 
     async def test_cortex_query_offload(self):
 
+        async def _hang_getNexsIndx(_):
```

Comment:
This definitely tests the hang once we have a valid link to make the RPC call with. Can we also add a test for `Proxy.getPoolLink / Proxy._initPoolLink` timing out; since we know that is another situation where this can wedge.

Reply:
added